records_mover/records/targets/table/target.py

```diff
@@ -74,29 +74,44 @@ def move_from_records_directory(self,
     def move_from_fileobjs_source(self,
                                   fileobjs_source: FileobjsSource,
                                   processing_instructions: ProcessingInstructions) -> MoveResult:
+        # TODO: Can this take in a loader so we don't have to do this assertion here?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker